lib/mailer.service.ts

```diff
@@ -105,9 +105,11 @@ export class MailerService {
 
   private verifyTransporter(transporter: Transporter, name?: string): void {
     const transporterName = name ? ` '${name}'` : '';
-    transporter.verify()
-      .then(() => this.mailerLogger.debug(`Transporter${transporterName} is ready`))
-      .catch((error) => this.mailerLogger.error(`Error occurred while verifying the transporter${transporterName}: ${error.message}`));
+
+    // wrap value in a promise to ensure then is always defined
+    new Promise(()=>transporter?.verify())
+            .then(() => this.mailerLogger.log(`Transporter${transporterName} is ready`))
+            .catch((error) => this.mailerLogger.log(`Error occurred while verifying the transporter${transporterName}}: ${error.message}`));
```

Comment:
Why change the log level (debug/error -> log)?

Reply:
True, the point here is to just wrap it in a promise and keep the default behavior